### PR TITLE
Map dashboard update to reflect current software

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

- Remove Map integration docs as integration has been removed in https://github.com/home-assistant/core/pull/113215
- Update Dashboard lists screenshot to include Map dashboard
- Update Map default dashboard docs to no longer refer to Map integration
- Remove/ update links to Map integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration others were adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/113215
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/getting-started/presence-detection.markdown [hunk 1]
```diff
@@ -18,14 +18,15 @@ It's also possible to run an app on your phone to provide detailed location info
...
 ### Zones
+
 ![Map with zones](/images/screenshots/badges-zone.png)
 
 {% term Zones %} allow you to name areas on a map. These areas can then be used to name the location a tracked user is, or use entering/leaving a zone as an automation {% term trigger %} or {% term condition %}. See [Zones integration](/integrations/zone/) page for more details like creating zones.
 
 <div class='note'>
-The map view will hide all devices that are home.
+The <a href="/dashboards/map/">map dashboard
+        </a>[map dashboard](/dashboards/map/) will hide all devices that are home.
```
Comment (klaasnicolaas) [inline source/getting-started/presence-detection.markdown]: Why not on a single line? 

If you keep a blank line between your content and the HTML tags, you could also place a URL in markdown style.

**Review by klaasnicolaas — DISMISSED** [review]

File: source/getting-started/presence-detection.markdown (hunk 1 above)
Comment (c0ffeeca7) [reply source/getting-started/presence-detection.markdown]: thank you for the catch!

**Review by frenck — DISMISSED** [review]
Let's add a redirect for `/integrations/map` to `/dashboards/map`.

That way, old URLs will still work

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**